components/Marketing/Trial.js

```diff
@@ -0,0 +1,211 @@
+import React, { useState, Fragment } from 'react'
+import { compose, graphql } from 'react-apollo'
+import gql from 'graphql-tag'
+import { css } from 'glamor'
+import { format } from 'url'
+import isEmail from 'validator/lib/isEmail'
+
+import { withSignIn } from '../Auth/SignIn'
+import Consents, { getConsentsError } from '../Pledge/Consents'
+import ErrorMessage from '../ErrorMessage'
+import SwitchBoard from '../Auth/SwitchBoard'
+import withMe from '../../lib/apollo/withMe'
+import withT from '../../lib/withT'
+import { TRIAL_CAMPAIGN } from '../../lib/constants'
+
+import { Button, Field, InlineSpinner, Interaction, colors } from '@project-r/styleguide'
+
+const { H1, P } = Interaction
+
+const styles = {
+  errorMessages: css({
+    color: colors.error,
+    marginTop: 40
+  }),
+  switchBoard: css({
+    marginTop: 40
+  }),
+  button: css({
+    marginTop: 40,
+    width: 180,
+    textAlign: 'center'
+  })
+}
+
+const REQUIRED_CONSENTS = ['PRIVACY', 'TOS']
+
+const REQUEST_ACCESS = gql`
+  mutation requestAccess($campaignId: ID!) {
+    requestAccess(campaignId: $campaignId) {
+      id
+      endAt
+    }
+  }
+`
+
+const toOnboarding = () => {
+  window.location = format({ pathname: '/einrichten', query: { context: 'trial' } })
+}
+
+const Trial = (props) => {
+  const { me, t } = props
+
+  const [consents, setConsents] = useState()
```

Comment:
IMO, initial state should always be explicit, in this case `useState(null)` if that's what you want.